ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -8554,6 +8554,9 @@ static int zend_jit_do_fcall(dasm_State **Dst, const zend_op *opline, const zend
 		unknown_num_args = 1;
 	}
 
+	/* TODO: Only do this for named args */
+	unknown_num_args = 1;
```

Comment:
Would it be possible to know if a `DO_*CALL` opcode corresponds to a call that contains named arguments by setting extended_value (or op2) when emitting the opcodes in Zend/zend_compile.c (e.g. to be used as a bit mask)

(E.g. anywhere such as zend_compile_call_common, zend_compile_func_cufa, etc)

Reply:
Right, I've done this now in https://github.com/php/php-src/pull/5357/commits/5cbfe4497b4aa8783585b0829324e72dbe6786a3.